packages/custom-webpack/src/schema.ext.json

```diff
@@ -7,7 +7,8 @@
         "path": {"type": "string", "description": "Path to the custom webpack configuration file"},
         "mergeStrategies": {"type": "object", "description": "Merge strategies per webpack config field"},
         "replaceDuplicatePlugins": {"type": "boolean", "description": "Flag that indicates whether to replace duplicate webpack plugins or not"}
-      }
+      },
+      "default": null
```

Comment:
Specifying default = null is redundant (it will be null anyways if not specified). 
So it's not necessary with the current implementation.